docs/guides/wifi.md

```diff
@@ -22,12 +22,12 @@ We now use a script which can help you set up Wi-Fi. Follow the instructions bel
 1. Click [here](../tools/wifi.sh) to download the script.
 2. Boot into macOS.
 3. Run this script there.
-4. The script will ask you to run some commands or the script itself in Linux after it gets executed successfully.
+4. The script will ask you to run some commands, or the script itself, in Linux after it gets executed successfully.
```

Comment:
Can you put the "in Linux after it gets executed successfully" clause first, i.e. something like "after the script is executed successfully, it'll ask you to boot to linux and then either run some commands or run the script itself a second time". This will keep that clause separate from the two listed options, increasing cohesion (imo).

Reply:
fbdf9e0